src/test/java/com/sfuentes/EmployeeTest.java

```diff
@@ -0,0 +1,69 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class EmployeeTest {
+
+  @Test
+  void amountSales_InvalidAmount_ExceptionThrown() {
+    double[] sales = {100, 100, 100, 100};
+
+    Exception exception = assertThrows(IllegalArgumentException.class, () -> {
+      Employee test = new Employee(1, sales);
+    });
+
+    String expectedMessage = "Incorrect amount of sales";
+
+    assertEquals(exception.getMessage(), expectedMessage);
```

Comment:
instead of using the variable expectedMessage please use the literal value "Incorrect amount of sales".